python/cuml/common/base.pyx

```diff
@@ -79,6 +79,30 @@ class Base:
         """
         self.handle = cuml.common.handle.Handle() if handle is None else handle
         self.verbose = verbose
+        
+    
+    def __repr__(self):
+        """
+        Pretty prints the arguments of a class using Sklearn standard :)
+        """
+        cdef list signature = inspect.getargspec(self.__init__).args
+        if signature[0] == 'self':
+            del signature[0]
+            
+        cdef dict state = self.__dict__
+        cdef str string = self.__class__.__name__ + '('
+        cdef str key
+        for key in signature:
+            try:
+                if type(state[key]) is str:
+                    string += "{}='{}', ".format(key, state[key])
+                else:
+                    string += "{}={}, ".format(key, str(state[key]))
+            except:
```

Comment:
Doing a second review, bare excepts are not ideal and also against PEP8 standard (E722), so this is a change that needs to happen among flake8/pep8 cleaning

Reply:
@dantegd  Oops lmao.... Forgot to check your comments..............

Reply:
Will fix this ASAP!!!